src/setup-dotnet.ts

```diff
@@ -66,6 +66,8 @@ export async function run() {
         installedDotnetVersions.push(installedVersion);
       }
       DotnetCoreInstaller.addToPath();
+    } else {
+      core.warning('No .NET versions were specified to be installed.');
```

Comment:
Wouldn't it be more informative to change the logging level in line 45 from `debug` to `info`? And add an `else` statement after the line 49 to incorporate another `info` log with text like that "global.json file isn't found. No version was specified to be installed."

Reply:
I can do it. Instead of this warning I added?

Reply:
I added an `info` (how about I make it a `warning` instead? It indicates the user might have done something wrong) if the `global.json` in the root wasn't found, and removed the `warning` I previously added. I kept the severity of the `debug` message you mentioned; better not emit any more diagnostics in the happy path. Let me know what you think.

Reply:
About keeping `debug` level, I think you're right - there is no need to spam customers if their intention is not to use `dotnet-version` and `global-json-file` inputs. As for the changing `info` to `warning` I think it'll be an overkill because some of our customers use action just for authentication (without need for SDK installation), so the `info` is enough in my opinion.

Reply:
Great. Can you approve again the CI?

Reply:
To solve issues with CI, just update the dotnet-installer scripts using: `npm run update-installers` and in the `workflow.yml` in the **test-proxy** job change this `image: datadog/squid:latest` to `image: ubuntu/squid:latest`.